Pipfile

```diff
@@ -5,6 +5,7 @@ name = "pypi"
 
 [packages]
 requests = "*"
+chardet= "*"
```

Comment:
This PR was already merged a few days ago and I'm sure this isn't important, but @royconst curious if you know how this made it into the pipfile in dbt-cta 🤔 did this slip in by accident or is it needed for something?